types.go

```diff
@@ -92,6 +175,52 @@ type (
 	}
 
 	errorCode int
+
+	//
+	// Fission-Environment interface. The following types are not
```

Comment:
Is this just about `FunctionLoadRequest`? For clarity it might be an idea to split these internal type definitions to a separate file/package.